src/tests/request.test.js

```diff
@@ -241,4 +239,28 @@ describe('/api/v1/requests', () => {
       res.body.should.be.an('object');
     });
   });
+  describe('GET /', () => {
+    it('should retrieve all requests made by the users', async () => {
+      const res = await chai.request(app)
+        .get(`${URL_PREFIX}/my-requests`)
+        .set('token', loginUser.body.data.token);
+
+      res.should.have.status(200);
+      res.body.data[0].should.have.property('destination');
```

Comment:
you could also assert the property value e.g `res.body.data[0].should.have.property('destination').eql('Lagos')` to be sure we are not getting a `null` value

Reply:
Ok